packages/fetchai/connections/http_client/connection.py

```diff
@@ -85,9 +85,12 @@ def __init__(
         )  # type: Optional[asyncio.AbstractEventLoop]  # pragma: no cover
         self.excluded_protocols = excluded_protocols
         self.is_stopped = True
-        logger.info("Initialised the HTTP client channel")
         self._tasks: Set[Task] = set()
 
+        self.logger = logger
+        # TODO logger at this point is the module-level one, not with the agent name.
```

Comment:
suggests we should pass logger during construction?

Reply:
here we are in the channel; that means we should pass the logger during construction also for the connection.

Reply:
@marcofavorito I don't think we should add TODO here in package;

Reply:
removed, thanks